src/js/boards.js

```diff
@@ -6,72 +6,201 @@
 // Also, please note that the targets in this list are ordered in the order they are returned by the above command. Please do not reorder to avoid churn.
 
 var BOARD_DEFINITIONS = [
+    {name: 'AG3X', identifier: 'AGX4', vcp: true},
+    {name: 'AG3X', identifier: 'AGX7', vcp: true},
+    {name: 'AIKONF4', identifier: 'AIK4', vcp: true},
     {name: 'AIR32', identifier: 'AR32', vcp: true},
-    {name: 'AIRHEROF3', identifier: 'AIR3', vcp: false},
```

Comment:
Changing these _will_ break flashing of these boards for firmware older than 3.3.0, as it relies on this list to figure out if a board has VCP or not.

Reply:
```
// This list has been extracted from the firmware source with:
// grep TARGET_BOARD_IDENTIFIER src/main/target/*/target.h | sed -n "s/^src\/main\/target\/\([^\/]*\).*#define TARGET_BOARD_IDENTIFIER \"\([^\"]*\).*$/    {name: '\1', identifier: '\2', vcp: true},/p" | sort
// and then manually setting vcp to true for boards that use VCP
// Also, please note that the targets in this list are ordered in the order they are returned by the above command. Please do not reorder to avoid churn.
```
Just updated accordingly the suggestion here. Did grep on current master.
😕 Maybe only the comment should be fixed.

Reply:
Yeah, I think the comment should be amended that this list is only needed for targets that were added before 3.3.0, and as such does not need to be updated any more.